src/pcapi/validation/routes/passwords.py

```diff
@@ -1,16 +1,34 @@
-import os
+from pcapi.connectors.api_recaptcha import get_token_validation_and_score
+from pcapi.models import ApiErrors
 
-from pcapi.connectors.api_recaptcha import validate_recaptcha_token
-from pcapi.models.api_errors import ApiErrors
 
+class ReCaptchaException(Exception):
+    pass
 
-# FIXME(cgaunet, 2020-11-17): change the required score once we have data on production environment
-RECAPTCHA_RESET_PASSWORD_MINIMAL_SCORE = float(os.environ.get("RECAPTCHA_RESET_PASSWORD_MINIMAL_SCORE", 0.0))
 
+class InvalidRecaptchaTokenException(ApiErrors):
+    def __init__(self):
+        super().__init__()
+        self.add_error("token", "Le token renseigné n'est pas valide")
 
-def validate_reset_password_token(token: str):
-    token_is_valid = validate_recaptcha_token(token, "resetPassword", RECAPTCHA_RESET_PASSWORD_MINIMAL_SCORE)
-    if not token_is_valid:
-        errors = ApiErrors()
-        errors.add_error("token", "Le token renseigné n'est pas valide")
-        raise errors
+
+def check_recaptcha_token_is_valid(token: str, original_action: str, minimal_score: float) -> None:
+    response = get_token_validation_and_score(token)
+    is_token_valid = response.get("success")
+
+    if not is_token_valid:
+        errors_found = response.get("error-codes", [])
+
+        if errors_found == ["timeout-or-duplicate"]:
+            raise InvalidRecaptchaTokenException()
+        else:
+            raise ReCaptchaException(f"Encountered the following error(s): {errors_found}")
+
+    response_score = response.get("score", 0)
+
+    if response_score < minimal_score:
```

Comment:
```suggestion
    if response_score <= minimal_score:
```
Sinon il faudra mettre 0.6 :)

Reply:
Discuté avec Cyril :)